test/studies/hpcc/common/bradc/unit/parRandom2-domblock64.chpl

```diff
@@ -13,18 +12,18 @@ var B: [ProblemSpace] real;
 var randStr1 = new RandomStream(314159265);
 var randStr2 = new RandomStream(314159265);
 
-forall (i,r) in (A.domain, randStr1) {
-  writeln("Doing iteration ", format("#####", i), " on locale ", here.id);
+forall (i,r) in zip(A.domain, randStr1) {
+  writeln("Doing iteration %##### on locale %i\n", i, here.id);
```

Comment:
yeah, %##### does look confusing. But the original format wouldn't have emitted a # sign, so I don't understand why the % is there? Doesn't this version generate an extra # (from %#) before printing the number?